int64 location_id
Snack[] snacks
